modules/standard/IO.chpl

```diff
@@ -9734,31 +9612,6 @@ proc fileReader.readBinary(ref data: [] ?t, endian: ioendian):int throws
   return nr;
 }
 
-@chpldoc.nodoc
-proc fileReader.readBinary(ref data: [] ?t, endian: ioendian):int throws
```

Comment:
Why are these two methods removed? I would think you would want to keep the methods and just delete the conditional (leaving it only with what was in the `== true` part).

Reply:
Oops, I deleted those while testing something and forgot to add them back. Thanks for catching that.